demo/init-container-failure.demo

```diff
@@ -0,0 +1,25 @@
+# Demo: inject fault to initContainer
+
+# Prepare manifest files
+make manifests
+clear
+
+# Create a cluster with kwok 
+kwokctl create cluster --config ./kustomize/stage/pod/chaos/pod-init-container-running-failed.yaml --config ./artifacts/stage-fast.yaml
+
+# Create and add 10 nodes.
+kwokctl scale node --replicas 10
+kubectl get nodes
+
+# Create a pod and inject an fault to initContainer
+grep failed demo/failed-pod.yaml
+kubectl apply -f demo/failed-pod.yaml
```

Comment:
I think it is better to create a ready pod first and trigger pod failure in the patch label to show this feature.

```
kwokctl scale pod --replicas 10
kubectl get pod
kubectl patch pod pod-000000 --type=merge --patch '{"metadata": { "labels":{"pod-container-running-failed.stage.kwok.x-k8s.io":"true"}}}'
kubectl get pod
```

Reply:
How about doing it in the demo with kwokctl and using failed-pod.yaml for the in-cluster demo, which is the  scenario in our use case?

Reply:
I think we should make a new folder for these demos related to specific examples.